src/main/java/club/sk1er/patcher/config/PatcherConfig.java

```diff
@@ -141,11 +141,33 @@
 
     @Property(
         type = PropertyType.SWITCH, name = "Disable Titles",
-        description = "Stop titles & subtitles from appearing.",
-        category = "Miscellaneous", subcategory = "Overlays"
+        description = "Stop titles from appearing.",
+        category = "Miscellaneous", subcategory = "Overlays", i18nSubcategory = "Titles"
     )
     public static boolean disableTitles;
 
+    @Property(
+        type = PropertyType.PERCENT_SLIDER, name = "Title Scale",
+        description = "Set the scale for titles.",
+        category = "Miscellaneous", subcategory = "Overlays", i18nSubcategory = "Titles"
```

Comment:
There's no need to use `i18nSubcategory` like this for new settings, and while it won't cause any problems, it's unnecessarily confusing.